src/ophys_etl/types/_roi.py

```diff
@@ -90,7 +90,7 @@ def __init__(self,
             raise ValueError("OphysROI.x0 must be an int; "
                              "you gave %s" % str(type(height)))
 
-        if valid_roi is None or not isinstance(valid_roi, bool):
+        if valid_roi is not None and not isinstance(valid_roi, bool):
```

Comment:
looked through the decrosstalk code and didn't see this used anywhere. I'm making it optional